LICENSE

```diff
@@ -214,18 +214,22 @@ Apache Software Foundation License 2.0
 3rdparty/dmlc-core
 
 
-BSD License
+BSD 2-clause License
 -----------
 
 3rdparty/picojson
+
+
+BSD 3-clause License
```

Comment:
Sorry I double checked, and find that the concurrentqueue.h is under BSD-2 (instead of 3)